src/state/actions/player.ts

```diff
@@ -127,20 +138,43 @@ export const playNextFromQueue = async () => {
   }
 }
 
+const handleLoadChapterForNowPlayingEpisode = async (item: NowPlayingItem) => {
+  await setPlaybackPosition(item.clipStartTime)
+  const nowPlayingItemEpisode = convertNowPlayingItemClipToNowPlayingItemEpisode(item)
+  await setNowPlayingItem(nowPlayingItemEpisode, item.clipStartTime || 0)
+  await PVTrackPlayer.play()
+  await loadChapterPlaybackInfo()
```

Comment:
Just a question. Do ALL these need to be awaits? Maybe some can happen in the background for performance

Reply:
Good point, I think setPlaybackPosition and loadChapterPlaybackInfo might not have to be awaits.